ta/Makefile

```diff
@@ -5,6 +5,8 @@ ifeq ($O,)
 $(error output path should be specified when calling this makefile)
 endif
 
+LDFLAGS=
```

Comment:
Can we use ‘-R’ or ‘--no-builtin-variables’ option to cancel all variables used by implicit rules?
If not, can you suggest a way how can I override CC, CPP, LD, AR etc by cross compile variables set by yocto?